tests/glob-std.rs

```diff
@@ -99,6 +99,22 @@ fn main() {
     mk_file("r/three", true);
     mk_file("r/three/c.md", false);
 
+    mk_file("dirsym", true);
+    mk_symlink_file("../r", "dirsym/link");
```

Comment:
Afaik windows distinguishes between dir and file symlinks.

Reply:
Why didn't the symlink call fail then.. :crying_cat_face:

Reply:
Ok, it was the relative path. Should be working now.